src/components/Dropdown/DropdownMenuItem/DropdownMenuItem.stories.tsx

```diff
@@ -0,0 +1,33 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { IconSize } from "@components/Icon/Icon";
+import { ReactComponent as Audio } from "@components/Icon/Svg/Audio.svg";
+import { Meta, Story } from "@storybook/react";
+import { Size } from "@utilities/enum";
+import DropdownMenuItem, { DropdownMenuItemProps } from "./DropdownMenuItem";
+
+export default {
+    title: "DropdownMenuItem",
+    component: DropdownMenuItem,
+    args: {
+        warning: false,
+        disabled: false,
+        onClick: { action: "onClear" },
+    },
```

Comment:
The `onClick`
```
    args: {
        //...
        onClick: { action: "onClear" },
    },
```
should be an `argTypes`
```
    argTypes: { 
      onClick: { action: "Item Clicked" },
    },
```
otherwise you get the following error `Uncaught TypeError: onClick is not a function`